functions/Export-DbaScript.ps1

```diff
@@ -154,19 +154,19 @@ function Export-DbaScript {
         $commandName = $MyInvocation.MyCommand.Name
         $prefixArray = @()
 
-        $appendToScript = $Append
+        # if the Append switch is used then ensure the scripting options are updated
+        $appendToScript = $false
+        if ($Append) {
+            $ScriptingOptionsObject.AppendToFile = $true
+            $appendToScript = $true
```

Comment:
The check for `$Append` should be removed and just set the options to true.

Reply:
This block of code needs to be removed, and line 158 can simply be changed to set it to `$true` since the parameter has been removed. Comment on line 157 can be deleted.

Reply:
This block of code needs to be removed, and line 158 can simply be changed to set it to `$true` since the parameter has been removed. Comment on line 157 can be deleted.

Reply:
This code is for Export-DbaScript, which needs to keep the Append param to support its own set of use cases. We are only removing the Append param from the Export-DbaInstance command.